src/execution/expression_executor/execute_operator.cpp

```diff
@@ -111,6 +111,35 @@ void ExpressionExecutor::Execute(const BoundOperatorExpression &expr, Expression
 		} else if (count == 1) {
 			result.SetVectorType(VectorType::CONSTANT_VECTOR);
 		}
+	} else if (expression_type == ExpressionType::OPERATOR_TRY) {
+		auto &child_state = *state->child_states[0];
+		try {
+			Execute(*expr.children[0], &child_state, sel, count, result);
+			return;
+		} catch (std::exception &ex) {
+			ErrorData error(ex);
+			auto error_type = error.Type();
+			if (!Exception::IsExecutionError(error_type)) {
+				throw;
+			}
+		}
+		SelectionVector selvec(1);
+		Vector intermediate(result.GetType(), 1);
+		for (idx_t i = 0; i < count; i++) {
+			selvec.set_index(0, sel ? sel->get_index(i) : i);
+			Value val;
+			try {
+				Execute(*expr.children[0], &child_state, &selvec, 1, intermediate);
```

Comment:
Do we need the `intermediate` here? Can't we directly pass in `result` with the specified offset?

Reply:
I don't think we can, that assumes the function will play nice and just write the value into the Vector data and leave it alone afterwards, but it could Slice, Flatten the Vector instead, or even allocate a new Vector and Reference to that one.

I could be wrong as I haven't tested this, but that sounds to me like it could break the expectations we have here.

Reply:
Also the `Execute` path doesn't have support for an offset as far as I can see

Reply:
That makes sense, agreed